frontend/models/product-list/server.ts

```diff
@@ -75,15 +76,17 @@ export async function findProductList(
    const algoliaApiKey = logSync('algolia:create key', () =>
       createPublicAlgoliaKey(ALGOLIA_APP_ID, ALGOLIA_API_KEY)
    );
-   const productListType = getProductListType(productList?.type);
 
    const ancestors = createProductListAncestors(parents);
+   const isPartsList =
+      productListType === ProductListType.AllParts ||
+      productListType === ProductListType.DeviceParts;
```

Comment:
Now that we're making the Algolia request after we fetch the product list, we don't need to pass `productListType` into `findProductList`, we can just pull it off the productList directly.

Reply:
https://github.com/iFixit/react-commerce/pull/1252/commits/f969a0e12764617e206205c42995d4d1156a8ae2